### ISSUE TITLE
Undefined constant "DS"

### ISSUE THREAD

**Issue by FynnZW** [bvdputte/kirby-logbook#4]
Using the plugin with PHP 8.3.12, and that line caused this error:

/site/plugins/kirby-logbook/index.php
``` 
require __DIR__ . DS . "classes" . DS . "Logbook.php"
```
 -> **Undefined constant "DS"**

I found a solution here: https://github.com/sylvainjule/kirby-colorextractor/issues/2
Adding this in Kirbys index.php solved it:
```php
if (!defined('DS')) define('DS', DIRECTORY_SEPARATOR);
include __DIR__ . '/../vendor/autoload.php';
```
Could this be done inside the plugin, to generally prevent this error?

**Comment by FynnZW** [comment]
Oh right, it's not version 1.0.5. 
I did a fresh install using composer: `composer require bvdputte/kirby-logbook`, but that installs 1.0.4:
```
Using version ^1.0 for bvdputte/kirby-logbook
...
  - Locking bvdputte/kirby-logbook (1.0.4)
```
It seems 1.0.5 is not published on packagist yet?

**Comment by bvdputte** [comment]
Ouch, you're right. Can you try again? I released 1.0.6 to fix this.

**Comment by FynnZW** [comment]
Perfect, it's solved now 👍 Thanks!
